### ISSUE TITLE
Schema missing in arango-orm, can't map Schemas

### ISSUE THREAD

**Issue by valentingregoire** [threatify/arango-orm#64]
When having a data structure like this: 
```json
{
    "first_name": "Valentin",
    "last_name": "Grégoire",
    "hobbies": [
        {
            "name": "Guitar",
            "type": "Music"
        }, {
            "name": "Speedcubing",
            "type": "Brain games"
        }
    ]
```
To map this with arango-orm, it would look something like this:
```python
from arango_orm import Collection
from arango_orm.fields import String, List, Nested
from marshmallow import Schema

class Hobby(Schema):
    name: str = None
    type: str = None


class Person(Collection):
    __collection__ = "persons"
    _key = String(required=True)
    first_name = String(required=True)
    last_name = String(required=True)
    hobbies = List(Nested(Hobby, required=True))
```

We use the `Nested` type because my data is known up front (structured data), else I would use `Dict` as stated in the docs.
As you can see, we don't have access to `Schema` directly from the `arango-orm` library like we do have on the `fields`. Might be interesting to map that.

So far, I can only map a list of objects by declaring it as a `Dict()`, and provide it as a `Python dictionary`, but not as an object.
```python
hobbies = Nested(Hobby, required=True)
```

**Comment by chris-green** [comment]
Hey @valentingregoire,

This is a issue I also hit, but https://github.com/threatify/arango-orm/pull/72 has been merged in which will allow for sub-collections schema to be passed in as nested values and instanciated upon load. 

This should resolve the issue

**Comment by kashifpk** [comment]
The features from @chris-green 's PR have been merged and released as 0.6.2. Closing issue.
